Userland/Libraries/LibWeb/IndexedDB/IDBFactory.cpp

```diff
@@ -55,13 +57,7 @@ JS::ThrowCompletionOr<JS::NonnullGCPtr<IDBOpenDBRequest>> IDBFactory::open(Strin
     Platform::EventLoopPlugin::the().deferred_invoke(JS::create_heap_function(realm.heap(), [&realm, storage_key, name, version, request] {
         HTML::TemporaryExecutionContext context(realm, HTML::TemporaryExecutionContext::CallbacksEnabled::Yes);
 
-        (void)storage_key;
-        (void)name;
-        (void)version;
-        (void)request;
-
-        // FIXME: 1. Let result be the result of opening a database connection, with storageKey, name, version if given and undefined otherwise, and request.
-        /*
+        // 1. Let result be the result of opening a database connection, with storageKey, name, version if given and undefined otherwise, and request.
         auto result = open_a_database_connection(realm, storage_key.value(), name, version, request);
```

Comment:
` LibWeb: Use open_a_database_connection in IDBFactory::open ` should be the commit to add this code, rather than uncommenting it :sweat_smile: